pkg/piperutils/FileUtils.go

```diff
@@ -47,3 +49,30 @@ func Copy(src, dst string) (int64, error) {
 	nBytes, err := io.Copy(destination, source)
 	return nBytes, err
 }
+
+// FindFiles ...
```

Comment:
As you can see from my comment below, proper documentation is justified. What format is "pattern" (regex versus glob) and what is matched (base-name).

Reply:
Yes, good example for having more docu ...